acceptance-tests/tests/src/test/resources/jsonrpc/eth/getBlockBy/test-cases/06_get_last_paris_block.json

```diff
@@ -0,0 +1,33 @@
+{
+  "request": {
+    "jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x1", false],"id":1
+  },
+  "response": {
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result" : {
+      "number" : "0x1",
```

Comment:
Would be a bit clearer if the block number for the last Paris block was 9. But since blocks, 2-8 are skipped it is actually the last one, just not what I expected going by the genesis config.

Reply:
Why 9?

Reply:
Aah, I mixed up block number and block time reading this. Yeah, block 2 is fine after all.